tracer/src/Datadog.Trace/Configuration/TracerSettings.cs

```diff
@@ -491,6 +491,20 @@ _ when x.ToBoolean() is { } boolean => boolean,
                                        .WithKeys(ConfigurationKeys.FeatureFlags.CommandsCollectionEnabled)
                                        .AsBool(false);
 
+            var defaultDisabledAdoNetCommandTypes = new string[] { "InterceptableDbCommand", "ProfiledDbCommand" };
+            var userDisabledAdoNetCommandTypes = config.WithKeys(ConfigurationKeys.DisabledAdoNetCommandTypes).AsString();
+
+            if (string.IsNullOrEmpty(userDisabledAdoNetCommandTypes))
+            {
+                DisabledAdoNetCommandTypes = new HashSet<string>(defaultDisabledAdoNetCommandTypes, StringComparer.OrdinalIgnoreCase);
+            }
+            else
+            {
+                var userSplit = TrimSplitString(userDisabledAdoNetCommandTypes, commaSeparator);
+                DisabledAdoNetCommandTypes = new HashSet<string>(userSplit, StringComparer.OrdinalIgnoreCase);
+                DisabledAdoNetCommandTypes.UnionWith(defaultDisabledAdoNetCommandTypes);
+            }
```

Comment:
I think you can refactor this a bit to always create the `HashSet` with `defaultDisabledAdoNetCommandTypes`, and then optionally add the `userDisabledAdoNetCommandTypes`. Not a big deal, I just think it flows a little better.

```suggestion
            DisabledAdoNetCommandTypes = new HashSet<string>(defaultDisabledAdoNetCommandTypes, StringComparer.OrdinalIgnoreCase);

            if (!string.IsNullOrEmpty(userDisabledAdoNetCommandTypes))
            {
                var userSplit = TrimSplitString(userDisabledAdoNetCommandTypes, commaSeparator);
                DisabledAdoNetCommandTypes.UnionWith(userSplit);
            }
```

Reply:
Very nice thanks!